test/bats/helpers/setup-and-teardown.bash

```diff
@@ -31,7 +31,7 @@ start_server() {
 }
 
 subscribe_to() {
-  background ${REPO_ROOT}/node_modules/.bin/ts-node "${REPO_ROOT}/src/debug/gqlsubscribe.ts" "ws://${GALOY_ENDPOINT}/graphqlws" "$(gql_file $1)" > .e2e-subscriber.log
+  background ${REPO_ROOT}/node_modules/.bin/ts-node "${REPO_ROOT}/test/helpers/servers/gql-subscribe.ts" "ws://${GALOY_ENDPOINT}/graphqlws" "$(gql_file $1)" > .e2e-subscriber.log
```

Comment:
I don't understand the reasoning behind this path.

Reply:
Do you mean aside from the explanation in the PR description?

Reply:
Reason is in the main description... or do you mean something else?